samples/model-builder/model_registry/upload_new_model_version_sample.py

```diff
@@ -0,0 +1,58 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from google.cloud import aiplatform
+
+
+# [START aiplatform_model_registry_upload_new_model_version_sample]
+def upload_new_model_version_sample(
+    model_id: str,
+    artifact_uri: str,
+    serving_container_image: str,
+    project: str,
+    location: str,
+):
+    """
+    Uploads a new version of a model to Vertex AI.
+    Args:
+        model_id: The ID of the model to upload a new version to. Parent resource name of the model is also accepted.
+        artifact_uri: The URI of the model artifact to upload.
+        serving_container_image: The name of the serving container image to use.
+        project: The project.
+        location: The location.
+
+    Returns:
+        The new version of the model.
+    """
+    # Initialize the client.
+    client = aiplatform.init(project=project, location=location)
```

Comment:
`client =` not part of recommended practice.
change `client.models.Model(model_id)` to `aiplatform.Model(name=model_id)`

Reply:
done